app/src/main/res/values-en/strings.xml

```diff
@@ -719,12 +719,12 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_buildingType_bungalow_description2">small detached house (summer house, holiday cottage, cabin …)</string>
     <string name="quest_buildingType_static_caravan">Mobile home</string>
     <string name="quest_buildingType_commercial">Commercial building</string>
-    <string name="quest_buildingType_commercial_generic_description">building where people work, shop or do any other commercial activities</string>
+    <string name="quest_buildingType_commercial_generic_description">building where people work, shop or do other commercial activities</string>
     <string name="quest_buildingType_industrial">Industrial building</string>
     <string name="quest_buildingType_industrial_description">e.g. a factory, workshop, car repair garage, …</string>
     <string name="quest_buildingType_office">Office building</string>
     <string name="quest_buildingType_retail">Shop(s)</string>
-    <string name="quest_buildingType_retail_description">including buildings for restaurants, cafés, big stores …</string>
+    <string name="quest_buildingType_retail_description">including buildings for restaurants, cafés, stores …</string>
```

Comment:
Nit: no space before ellipses in English. This seems to occur in several places, so it might be worth a followup PR.

Reply:
I saw this but left them alone thinking it was some kind of formatting thing.

Reply:
https://github.com/streetcomplete/StreetComplete/issues/4032